config/routes.rb

```diff
@@ -32,7 +32,11 @@
         member { post :move }
         resources :cards, except: [:index] do
           member { post :move }
-          resources :revisions, only: [:index, :show]
+          resources :revisions, only: [:index, :show] do
+            collection do
```

Comment:
See comment above about using a single route vs a sidebar route for every type of item.